### PR TITLE
magnitude dimension estimator with tests

### PR DESCRIPTION
**Pull request by kmyim**
Added the magnitude dimension estimator as proposed in "Metric Space Magnitude and Generalisation in Neural Networks".

### CODE REVIEW

File: skdim/id/_Mag.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        np.array with the log(magnitude(t)) values.
+    reg_: sklearn.linear_model.LinearRegression
+        Regression object used to fit line to log tM vs log t
+    """
+    def __init__(self, ts = np.linspace(0.5,5.5,21), metric = 'euclidean'):
+        self.ts = ts
+        self.metric = metric 
+
+    def fit(self, X, y=None):
+        """A reference implementation of a fitting function.
```
Comment (Malinon) [inline skdim/id/_Mag.py:35]: I think that this line can be removed.

**Review by Malinon — COMMENTED** [review]
It looks good. It can be merged after fixing the comment line. Maybe in future we should add more tests for this method.
